lib/app/features/auth/views/pages/recover_user_twofa_page/components/twofa_options_selector_button.dart

```diff
@@ -49,6 +51,7 @@ class TwoFaOptionsSelectorButton extends StatelessWidget {
       ),
       opened: isOpened.value,
       onPressed: () {
+        if (!enabled) return;
```

Comment:
better set onPressed as `null` instead of return;

Reply:
in the current implementation of `Button.dropdown`, the `onTap` is required, so we can't set it to null.